src/items/equipment/excavation_drill_helical.json

```diff
@@ -102,24 +87,45 @@
         {
           "formula": "2d6",
           "operator": "",
-          "types": {}
+          "types": {
```

Comment:
Do these drills deal untyped damage?

Reply:
It does not, it deals piercing damage, didn't see this issue, I will fix it.